python/pyspark/ml/tuning.py

```diff
@@ -143,6 +145,8 @@ def getEvaluator(self):
 
 class CrossValidator(Estimator, ValidatorParams):
     """
+    .. note:: Experimental
```

Comment:
Is there more detailed Scaldoc we can copy across? If not then this class should definitely have more detailed doc across Scala and Python.